recipe/build.sh

```diff
@@ -1,5 +1,20 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-make -f unix/Makefile generic CC=$CC CPP=$CPP
-mkdir -p $PREFIX/bin
-mv zip $PREFIX/bin
+set -eux
+
+export CFLAGS="${CFLAGS} -DLARGE_FILE_SUPPORT -DZIP64_SUPPORT" \
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+
+mkdir -p bzip2
+
+# patch in default conda-forge compiler flags
+sed -i "s|^CFLAGS_NOOPT =|CFLAGS_NOOPT = $CFLAGS $CPPFLAGS |" unix/Makefile
+sed -i "s|^LFLAGS1=''|LFLAGS1='$LDFLAGS'|" unix/configure
+
+# apply security patches from fedora
+patch -p1 -i zip-3.0-currdir.patch
```

Comment:
i guess you are trying to not vendor their patches.

ok. but I do find that it is often easier to vendor the patches, and let conda do it.

Reply:
How about the approach from  c653458 :
- download the patches with SHA first
- download the source
  - let `patches:` do the actual patching

Reply:
great thanks.